api/managementpb/backup/artifacts.proto

```diff
@@ -18,6 +18,8 @@ enum BackupStatus {
   BACKUP_STATUS_ERROR = 5;
   BACKUP_STATUS_DELETING = 6;
   BACKUP_STATUS_FAILED_TO_DELETE = 7;
+  BACKUP_STATUS_RETENTION_IN_PROGRESS = 8;
+  BACKUP_STATUS_FAILED_RETENTION = 9;
```

Comment:
I think we can something else instead of `RETENTION` here, maybe `CLEANUP`? Because retention - it's how artifacts keeping organized.